spring-cloud-dataflow-server/src/main/java/org/springframework/cloud/dataflow/server/single/DataFlowServerApplication.java

```diff
@@ -45,6 +47,7 @@
 		CloudFoundryDeployerAutoConfiguration.class,
 		KubernetesAutoConfiguration.class
 })
+@ComponentScan(basePackageClasses = {AppRegistrationDao.class})
```

Comment:
If this is the default server, then why would this not have a `ComponentScan` before?

Reply:
There is another class that has other annotations but when I ran the app it failed to start, adding this here made it work. Still checking this.